src/Aquifer.API/Endpoints/Marketing/Unsubscribe/Validator.cs

```diff
@@ -0,0 +1,12 @@
+﻿using FastEndpoints;
+using FluentValidation;
+
+namespace Aquifer.API.Endpoints.Marketing.Unsubscribe;
+
+public class Validator : Validator<Request>
+{
+    public Validator()
+    {
+        RuleFor(x => x.UnsubscribeId).NotEmpty().Length(64);
```

Comment:
Is having Length validated so specifically a good idea? Would something like MaxLength(100) or something be better so that the string length wouldn't be obvious?

Reply:
I'm not sure it would matter having this here since it's just doing a lookup in the DB either way. I don't think it's bad if people know it's 64 chars long but I also don't know if it's worth having the validation 🤷

Reply:
If someone wants to be obnoxious they'll see right away to just send a 64 character string, and then keep at it. In the meantime, if we ever decided to change the unsubscribe id to 128 characters or something, we'd have to remember to update this. So I'd just remove it.